.github/workflows/cmake.yml

```diff
@@ -34,17 +47,29 @@ jobs:
         sudo apt-get install clang-format-11 libstdc++-10-dev libmpfr-dev libmpc-dev ninja-build npm
       # If we ever do any qemu-emulation on Github Actions, we should re-enable this:
       # e2fsprogs qemu-system-i386 qemu-utils
-    - name: Install prettier
-      run: sudo npm install -g prettier
+      if: ${{ runner.os == 'Linux' }}
     - name: Use GCC 10 instead
       run: sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 60 --slave /usr/bin/g++ g++ /usr/bin/g++-10
+      if: ${{ runner.os == 'Linux' }}
+
+    - name: Install MacOS dependencies
+      run: brew install coreutils ninja
+      if: ${{ runner.os == 'macOS' }}
+
+    - name: Install js dependencies
+      run: sudo npm install -g prettier
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 requests
```

Comment:
```suggestion
        python3 -m pip install --upgrade pip
        pip3 install flake8 requests
```

Reply:
The default `setup-python@v2` action will set `python` to `python3.9`. Should I make that step more explicit, and/or still go ahead with this change? i.e. pip will be pip3 as well

Reply:
ok great, that's fine like this then :)

Reply:
I added a few comments to clarify this.